WordPress/Resources/ar.lproj/Localizable.strings

```diff
@@ -9497,6 +9494,9 @@ translators: %s: Select control option value e.g: \"Auto, 25%\". */
 /* Sentence to justify why the app asks permission from the user to access their Media Library. */
 "infoplist.NSPhotoLibraryUsageDescription" = "لإضافة صور أو مقاطع فيديو إلى مقالاتك.";
 
+/* Footer text for Invite Links section of the Invite People screen. */
+"invite_people_invite_link_footer" = "استخدم هذا الرابط لضم أعضاء فريقك من دون دعوتهم واحدًا تلو الآخر. سيتمكن أي شخص يزور عنوان URL هذا من التسجيل في مؤسستك، حتى إذا تلقى الرابط من شخص آخر، لذا تأكَّد من مشاركته مع أشخاص موثوق بهم.";
```

Comment:
This pair of removed and added key is a result of the change from [this commit](https://github.com/wordpress-mobile/WordPress-iOS/pull/18425/commits/8ac6584469211cf5206b853b752c5afc119b3f84). Same for all the other locales.